Essentials/src/com/earth2me/essentials/commands/EssentialsLoopCommand.java

```diff
@@ -19,27 +19,31 @@ public EssentialsLoopCommand(String command) {
         super(command);
     }
 
-    protected void loopOfflinePlayers(final Server server, final CommandSource sender, final boolean multipleStringMatches, boolean matchWildcards, final String searchTerm, final String[] commandArgs) throws PlayerNotFoundException, NotEnoughArgumentsException, PlayerExemptException, ChargeException, MaxMoneyException {
```

Comment:
Not sure about this; sure, this is a lot of exceptions in the method signature, but this encourages throwing whatever you want and not caring about what you catch.